modules/FindHarfBuzz.cmake

```diff
@@ -61,3 +62,5 @@ if(NOT TARGET HarfBuzz::HarfBuzz)
         IMPORTED_LOCATION ${HARFBUZZ_LIBRARY}
         INTERFACE_INCLUDE_DIRECTORIES ${HARFBUZZ_INCLUDE_DIR})
 endif()
+
+set(HARFBUZZ_LIBRARIES "${HARFBUZZ_LIBRARY}")
```

Comment:
From my investigation: once you create a string with spaces in it it, it will stay so in all further variable expansions, unless you explicitly replace spaces with `;`.